app/javascript/components/student/tracks-list/TagOptionList.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { TagOption } from './TagOption'
+
+export function TagOptionList({ options, onSubmit, value, setValue }) {
```

Comment:
I didn't immediately understand what `value` represented. We do use it in several other places, so I can understand why we're using it. Just wanted to put out an alternative:

```suggestion
export function TagOptionList({ options, onSubmit, selectedTags, setSelectedTags }) {
```